### ISSUE TITLE
[Feat]: Field Label 'for' attribute should accept "DOM Token List"

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#2438]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Field Label

### Description of the requested feature

Allow the `for` attribute to accept multiple IDs so that the Field Label can label multiple elements.

### Implementation notes or ideas

Should be able to just split on `" "` and then run the current code path on a map.

**Comment by nickschaap** [comment]
Curious what the expected behavior here is for when multiple for targets are specified and the label is clicked? Which target should get focused if any?

**Comment by kieransukachevin** [comment]
I implemented the suggested changes and also changed the behavior to cycle through focusing on the targets when the field label is clicked, but I am also curious what the expected behavior is.

**Comment by Westbrook** [comment]
Looking into `for` on a `<label>`, seems like this was a bridge too far, as far as a request: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#for
